tensorboard/webapp/plugins/plugins.container.ts

```diff
@@ -54,10 +49,10 @@ const lastLoadedTimeInMs = createSelector(
 
 @Component({
   selector: 'plugins',
-  templateUrl: './plugins.component.html',
-  styleUrls: ['./plugins.component.css'],
+  template: '<div #plugins class="plugins"></div>',
+  styles: ['.plugins { height: 100%; }', 'iframe { border: 0; }'],
```

Comment:
Might be naive question: Why inlining the CSS instead of importing a .css file like before?

Reply:
If I may guess, I presume that is because we would like to avoid unnecessary template extraction and building step of Angular by just providing the content of CSS and HTML directly here. Not sure if there is any build time performance gain when using AOT but it should help with JIT.

Reply:
I was just following our rule of thumb that inlining is easier when there are only a few lines of code.  Additional files carry the burden of maintaining BUILD, directory listing bloat, etc.  That's worth it only when they have enough content that it'd be awkward to inline.